src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/OrderRefusalTest.java

```diff
@@ -25,7 +30,7 @@ public void setUp() throws Exception {
     public void shouldCreateOrderRefusalFromJson() {
         assertThat(order.getOrderRefusalAfterText(), is("afterText"));
         assertThat(order.getOrderRefusal(), hasItems("Other"));
-        assertThat(order.getOrderRefusalDate(), is(Date.valueOf("2003-02-01")));
+        assertThat(order.getOrderRefusalDate(), is(parseDate("2003-02-01")));
```

Comment:
I think I'd put a .toString in front of the date and compare the strings.

Reply:
It was returning:

```
Expected : is <Sat Feb 01 00:00:00 CET 2003>
Actual   :<Sat Feb 01 01:00:00 CET 2003>
```

Reply:
hum. weird...
even with the java.util.Date?
Documentation says it should return in "yyyy-MM-dd" format.
https://docs.oracle.com/javase/8/docs/api/java/sql/Date.html#toString--

Reply:
Ignore me. I was looking at the java.sql one.

Reply:
Maybe ".getInstant().toString()" will work. If you think it's worth a try.

Reply:
```order.getOrderRefusalDate().toInstant().toString()```

returns:
`2003-02-01T00:00:00Z`

so timezone issue is still there :/